src/content/docs/reference/stages.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Stages
+description: A stage is a separate image build flow executed in parallel with the main build.
+---
+
+:::note
+This feature is experimental and currently only available when using `use_unstable_cli` option in the [GitHub Action](/reference/github-action/) or using the `stages` feature when compiling.
```

Comment:
Isn't the stages feature in CLI v0.8.10? @gmpinder

Reply:
No it's still experimental status, though I've not had issues with it from my own use. I'd like some more folks to try using it before we fully release it if possible

Reply:
> some clarification on how I'm handling brand new features at the moment. I'm putting them under their own feature flags. These feature flags are compile time features meaning that you have to enable them using the --features arg in cargo in order for it to be enabled. For all main branch builds I enable --all-features flag, so that the use_unstable_cli input for the action will have these features turned on. After some time, testing, or a new major/minor version bump, I add that feature to the list of default features which would then make it available in the standard releases 
> For the stages feature, it doesn't have any breaking changes per se, but I do want some users to be able to try it out and give some feedback
> That way if we have to make a fundemental change to it before wider release, we can do that
> I've been rocking the stages feature for a while and I'm liking it so far, but I'm only one person. Adding the documentation in with the caveat that it's only available when using the unstable cli will essentially let the user know that it could change in the future
> It will also allow users that do want to try it to be able to see how to currently use it
> We got one issue from a user that was already starting to use it, but they wanted documentation on it so they knew what all they could do
> That's why I'm trying to push for this documentation to go out for something that hasn't been fully released quite yet

My reasoning I put in Discord